src/main/resources/ai/privado/rulevalidator/schema/policies.json

```diff
@@ -114,6 +114,18 @@
                   ],
                   "pattern": "^.*$"
                 }
+              },
+              "sinkFilters": {
+                "$id": "#root/policies/items/dataFlow/sinkFilters",
+                "title": "SinkFilters",
+                "type": "object",
```

Comment:
Also include properties as well. I see it is missing for `sourceFilters`, add there as well